src/Admin/TerritorialCouncil/TerritorialCouncilMembershipAdmin.php

```diff
@@ -80,6 +112,35 @@ protected function configureDatagridFilters(DatagridMapper $filter)
                 'label' => 'Date',
                 'field_type' => DateRangePickerType::class,
             ])
+            ->add('isInPoliticalCommittee', CallbackFilter::class, [
+                'show_filter' => true,
+                'label' => 'Est dans le CoPol ?',
+                'field_type' => ChoiceType::class,
+                'field_options' => [
+                    'choices' => [
+                        'common.all' => null,
+                        'global.yes' => true,
+                        'global.no' => false,
+                    ],
+                ],
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    if (null !== $value['value']) {
+                        if (!\in_array('adherent', $qb->getAllAliases())) {
+                            $qb
+                                ->leftJoin("$alias.adherent", 'adherent')
+                                ->leftJoin('adherent.politicalCommitteeMembership', 'pcMembership')
+                            ;
+                        }
+
+                        $condition = true === $value['value'] ? 'NOT' : '';
+                        $qb->andWhere("pcMembership.id IS $condition NULL");
```

Comment:
this should be into the if above non ? if we don't enter the if, pcMembership is not defined

Reply:
No, the `if` above exists to `adherent` and `pcMembership` aliases, if they have not yet been added.